apps/web/src/locales/en/messages.json

```
@@ -3593,7 +3593,7 @@
     "message": "This item has old file attachments that need to be fixed."
   },
   "attachmentFixDesc": {
-    "message": "This is an old file attachment the needs to be fixed. Click to learn more."
+    "message": "This attachment uses outdated encryption. Select Fix to download, re-encrypt, and re-upload the attachment."
```

Should Fix be in quotes here?

```suggestion
    "message": "This attachment uses outdated encryption. Select 'Fix' to download, re-encrypt, and re-upload the attachment."
```